include/seqan3/range/container/small_string.hpp

```diff
@@ -80,7 +80,7 @@ class small_string : public small_vector<char, capacity_ + 1>
      * No-throw guarantee.
      */
     template <size_t N>
-    explicit constexpr small_string(char const (&_lit)[N]) noexcept : small_string{}
+    constexpr small_string(char const (&_lit)[N]) noexcept : small_string{}
```

Comment:
we need implicit construction from string literals to use this in template non-type argument contexts.